.pullapprove.yml

```
@@ -1221,6 +1221,23 @@ groups:
         - IgorMinar
         - mgechev
 
+  # =========================================================
+  #  Docs: All
+  #
+  # Ensures the `comp: docs` label is applied to aio doc changes
+  # =========================================================
+  docs-all:
+    type: optional
+    conditions:
+      - >
+        contains_any_globs(files, [
+          'aio/content/guide/**',
+          'aio/content/examples/**'
```

We should also include `aio/content/images/guide/`:

```suggestion
          'aio/content/examples/**',
          'aio/content/images/guide/**'
```